src/api/integrations/companies-house/index.ts

```diff
@@ -26,6 +26,13 @@ const companiesHouse = {
         },
       });
 
+      if (response.status === 404) {
```

Comment:
wondering if in the future, these codes should be constants

Reply:
Would be good, can we look into this later please. Quite a few instances of status checking, not sure what we would call them either. I.e  i don't believe eslint will allow us to create a constant starting with a number.